packages/conf-openblas/conf-openblas.0.2.1/opam

```diff
@@ -35,6 +45,9 @@ x-ci-accept-failures: [
   "oraclelinux-7"
   "oraclelinux-8"
 ]
```

Comment:
Needs a dependency on `conf-pkg-config` somewhere here to work correctly.
```suggestion
]
depends: [
  "conf-pkg-config"
]
```

Reply:
```suggestion
]
depends: [
  "conf-pkg-config" {os = "macos" & os-distribution = "homebrew"}
]
```

Reply:
I think I would restrict it to where it is really needed. With macports I never had any troubles